modal/client.py

```diff
@@ -426,5 +430,8 @@ async def unary_stream(
         request,
         metadata: Optional[Any] = None,
     ):
+        if self.client._snapshotted:
+            logger.debug(f"refreshing client after snapshot for {self._wrapped_method_name}")
+            self.client = await _Client.from_env
```

Comment:
from_env is a method that needs to be called (`()`) so this probably crashes 😬  - can we add a test that covers this case?